### PR TITLE
404

### PR DESCRIPTION
**Pull request by mbouzi**
- creation of 404 page at "/404

### CODE REVIEW

File: layouts/partials/404/404.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+        <div class="spacer20"></div>
+        <div class="p404__msg">
+          {{ .Params.content}}
+        </div>
+        <div class="spacer20"></div>
+        <a href="/">
+          <h2 class="p404__cta">
+            {{ .Params.cta}}
+            <img class="mb-1" src="/img/blue-carrot.svg" height="13" width="13"/>
```
Comment (boshevski) [inline layouts/partials/404/404.html]: The arrow should be before the text and to be pointing to the left (back)

File: layouts/partials/404/404.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<div class="lite-gray-bg container-fluid p404">
+  <div class="container">
+    <div class="row p404__content-wrapper d-flex justify-content-center">
+      <div class="spacer125"></div>
+      <div class="col-lg-5 p404__content">
```
Comment (boshevski) [inline layouts/partials/404/404.html]: The content is not aligned left with the logo. Use class "col-lg-6"

File: layouts/partials/404/404.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        </div>
+        <div class="spacer20"></div>
+        <a href="/">
+          <h2 class="p404__cta">
+            {{ .Params.cta}}
+            <img class="mb-1" src="/img/blue-carrot.svg" height="13" width="13"/>
+          </h2>
+        </a>
+      </div>
+      <div class="col-lg-4 offset-2 p404__mascot">
```
Comment (boshevski) [inline layouts/partials/404/404.html]: Change "col-lg-4" to "col-lg-6" and remove "offset-2" from the classes.

File: content/404/404.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+heading: "Oops! That's not a thing..."
+content: "The page you're trying to access is either broken, or doesn't exists."
+image: "/img/404-mascot.png"
```
Comment (boshevski) [inline content/404/404.md]: Image is low quality. Try to use a higher resolution version.

File: layouts/404/single.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    {{partial "head" .}}
+</head>
+<body class="p404-page page">
+    {{ partial "banner" .}}
+    <div  class="container-fluid pt-3 pb-1 storj-blu-bg">
+        <div class="container storj-blu-bg">
+        {{ partial "nav"  .}}
```
Comment (boshevski) [inline layouts/404/single.html]: Looking at the mockups on figma:
https://www.figma.com/file/g7doUVwCahjA5YHJNQnNd7A5/Tardigrade.io?node-id=2403%3A0
The navbar is using a white background with blue logo, links and button.

Comment (boshevski) [reply layouts/404/single.html]: Like the navbar on the legal pages.

File: layouts/404/single.html [hunk 6]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    {{partial "head" .}}
+</head>
+<body class="p404-page page">
+    {{ partial "banner" .}}
+    <div  class="container-fluid">
```
Comment (boshevski) [inline layouts/404/single.html]: No need for this parent div.

File: layouts/404/single.html [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+        <div class="container">
```
Comment (boshevski) [inline layouts/404/single.html]: Same here, no need for this one also.

Comment (boshevski) [reply layouts/404/single.html]: It messes up the alignment
logo

File: layouts/404/single.html [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+        {{ partial "secondNav"  .}}
+        </div>
+        <div class="container-fluid pt-3 d-xs-block d-lg-none storj-blu-bg"></div>
```
Comment (boshevski) [inline layouts/404/single.html]: Don't need this one as well. It shows as blue block on mobile:

blue

File: layouts/404/single.html (hunk 7 above)
Comment (mbouzi) [reply layouts/404/single.html]: thanks for pointing these out!

**Pull request merged**